bot/constants.py

```diff
@@ -40,6 +42,10 @@ class Channels(NamedTuple):
     dev_branding = int(os.getenv("CHANNEL_DEV_BRANDING", 789193817051234306))
 
 
+class Tokens(NamedTuple):
+    wolfram_id = os.getenv("WOLFRAM_TOKEN") or None
```

Comment:
```suggestion
    wolfram_token = os.getenv("WOLFRAM_TOKEN") or None
```
Seems more appropriate.

Reply:
as you have made the correction yourself, can i commit it?

Reply:
You would need to change this at the other places too, since you are importing this, so I would say no.